src/components/enter-password/enter-password-controller.ts

```diff
@@ -187,6 +187,8 @@ export function enterPasswordPost(
       }
     }
 
+    req.session.user.isSignInJourney = true;
```

Comment:
no use of the support2hrLockout() switch here, is it not required?

Reply:
the session variables for the journeys are only used now in the 2hr screens, I didn't put them them in a flag as they wouldn't impact anything else (as they are new and not used anywhere else) and they might me useful in other tickets